File: 프로그래머스/Level1/가운데 글자 가져오기.py
# https://programmers.co.kr/learn/courses/30/lessons/12903

def solution(s):
    len_s = len(s)
    if len_s % 2 == 0:
        return s[len_s // 2 - 1 : len_s // 2 + 1]
    else:
        return s[len_s // 2]

File: 프로그래머스/Level1/가운데 글자 가져오기.js
// https://programmers.co.kr/learn/courses/30/lessons/12903

const solution = (s) =>
	s.length % 2 === 0
		? s[parseInt(s.length / 2 - 1)] + s[parseInt(s.length / 2)]
		: s[parseInt(s.length / 2)];
